docs/extend/extend-apiml/api-mediation-onboarding-with-spring-boot-enabler

```diff
@@ -0,0 +1,502 @@
+
+2. [Configuring your project](#configuring-your-project)
+
+    * [Gradle guide](#gradle-guide)
+    * [Maven guide](#maven-guide)
+
+3. [Configuring your Spring Boot based service to onboard with API ML](#configuring-your-spring-boot-based-service-to-onboard-with-api-ml)
+
+4. [Registering and unregistering your service with API ML](#registering-and-unregistering-your-service-with-api-ml)
+
+5. [Adding API documentation](#adding-api-documentation)
+
+6. (Optional) [Validating your API service discoverability](#validating-the-discoverability-of-your-api-service-by-the-discovery-service)
+
+
+## Selecting a Spring Boot Enabler
+
+Add to your project build configuration a dependency on Spring Enabler version corresponding to the Spring Boot version, that you use for the whole project:
+- onboarding-enabler-spring-v1
+- onboarding-enabler-spring-v2
+
+**Note:** The process of onboarding an API service is the same for both Spring Boot enabler versions.
+
+## Configuring your project
+
+Use either _Gradle_ or _Maven_ build automation systems to manage your project builds.
+
+**Note:** You can download the selected enabler artifact from the Giza Artifactory or  if you decide to build the API ML from source, you are required to publish the enabler artifact to your Artifactory.
+ Publish the enabler artifact by using the _Gradle_ tasks provided in the source code.
+
+### Gradle build automation system
+Use the following procedure to use _Gradle_ as your build automation system.
+
+**Follow these steps:**
+
+1. Create a `gradle.properties` file in the root of your project if one does not already exist.
+
+2. In the `gradle.properties` file, set the URL of the specific Artifactory containing the _SpringEnabler_ artifact. Provide the corresponding credentials to gain access to the Maven repository.
+
+    If you are using the Giza Artifactory, use the credentials in the following code block:
+
+    ```ini
+    # Repository URL for getting the enabler-java artifact
+    artifactoryMavenRepo=https://gizaartifactory.jfrog.io/gizaartifactory/libs-release
+
+    # Artifactory credentials for builds:
+    mavenUser=apilayer-build
+    mavenPassword=lHj7sjJmAxL5k7obuf80Of+tCLQYZPMVpDob5oJG1NI=
+    ```
+
+3. Add the following _Gradle_ code block to the `repositories` section of your `build.gradle` file:
+
+    ```gradle
+    repositories {
+        ...
+
+        maven {
+            url artifactoryMavenRepo
+            credentials {
+                username mavenUser
+                password mavenPassword
+            }
+        }
+    }
+    ```
+4.  In the same `build.gradle` file, add the necessary dependencies for your service. If you use the _SpringEnabler_ from the Giza Artifactory, add the following code block to your `build.gradle` script:
+
+<font color="red">TODO#FindOut what name will be SE published under</font>
+
+```gradle
+implementation "com.ca.mfaas.sdk:mfaas-integration-enabler-java:$zoweApimlVersion"
```

Comment:
Every `com.ca.*` is now changed to `org.zowe.apiml` so we need to update also this markdown.